packages/builder/tsconfig.json

```diff
@@ -6,7 +6,6 @@
     "sourceMap": true,
     "baseUrl": ".",
     "paths": {
-      "@budibase/types": ["../types/src"],
```

Comment:
I believe the paths here allowed us to references sources directly, rather than relying in a `yarn build` before the differences would be picked up locally. When we add esbuild integration (currently in progress) we may be able to remove this. @adrinr might be able to give a better explanation. 

With that said I'm not sure why we have the `backend-core` dependencies here, they can probably be removed.